_includes/misc/purchase-premium-plugins.md

```diff
@@ -0,0 +1,51 @@
+{% if extensionType != 'Packs' %}
+{% assign extensionType = 'plugin' %}
+{% endif %}
+{% if pluralExtensionType == true %}
+{% assign isAre = 'are' %}
+{% else %}
+{% assign isAre = 'is' %}
+{% endif %}
+### Getting started with TinyMCE {{pluginname}}
```

Comment:
Looking at the rendered pages, should this be a h2 instead?

Reply:
Hm. In some pages it looks right and in others it looks wrong. Maybe need to go through the premium plugin pages and make the heading types more consistent?

Reply:
Also if you're using a variable here, maybe you should use `{{site.productname}}` instead of `TinyMCE`?

Reply:
@metricjs changed to h2, should be fine like that.
@lnewson I think it's more variable than heading now.